src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,182 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
```

Comment:
Do you really need to initialize fields with default values? But you don't need to fix it if you think 'yes'. I leave it to the mentors, maybe it's like good tone to do it this way. I'm just pointing this out cause it's an open question for me =)

Reply:
@ystankevych actually there is no sense to init the fields when creating the list instance

default value for "object" fields - null
default value for primitive int - 0

this is suitable for us :)